DigitalLearningSolutions.Web/Controllers/LearningPortalController/LearningPortalController.cs

```diff
@@ -12,6 +12,7 @@ namespace DigitalLearningSolutions.Web.Controllers.LearningPortalController
     using Microsoft.Extensions.Logging;
 
     [Authorize(Policy = CustomPolicies.UserOnly)]
+    [SetDlsSubApplication(nameof(DlsSubApplication.LearningPortal))]

```

Comment:
The "SetDlsSubApplication" just sets a piece of ViewData with the key "DlsSubApplication". This is then used in the layout to automatically determine what nav bar, header, etc. to use. It was designed so that we no longer need to specify all those bits of ViewData (except the title) on each view, but specifying those bits overwrites whatever is automatically determined from that attribute's ViewData - this meant we did not need to refactor every single page, so this is safe to implement.

Reply:
Have you tested it doesn't change anything? I know there are some pages on the Learning Portal that aren't meant to have nav bars, though I don't know whether they're in this controller or not.

Reply:
Ah, I was not aware of that. Do we have a list of those pages anywhere? Or shall I just remove the attribute from the controller?

Reply:
I _think_ it's the LearningMenu stuff so you might be alright. If you launch a course or self assessment then the nav bar disappears.
But I wouldn't add this attribute unless you've tested all the endpoints in the controller.